### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by Hanope**

### CODE REVIEW

File: eureka-core/src/main/java/com/netflix/eureka/registry/InstanceRegistry.java [hunk 1]
```diff
@@ -80,7 +80,7 @@ boolean deleteStatusOverride(String appName, String id, InstanceStatus newStatus
 
     int getNumOfRenewsPerMinThreshold();
 
-    int isBelowRenewThresold();
+    int isBelowRenewThreshold();
```
Comment (troshko111) [inline eureka-core/src/main/java/com/netflix/eureka/registry/InstanceRegistry.java:83]: This cannot be renamed as it's a part of a public contract (an interface method of a public type).

**Comment by spencergibb** [comment]
Maybe a new method that defaults to calling the misspelling?

**Comment by troshko111** [comment]
> Maybe a new method that defaults to calling the misspelling?

I normally see these corrected in the doc string / comment only, but either way works.

**Comment by Hanope** [comment]
> LGTM, but please revert the public interface method rename.

Okay, I revert files.

**Pull request merged**